src/framework/creg.hpp

```diff
@@ -178,13 +178,20 @@ void ClassicalRegister::apply_bfunc(const Operations::Op &op) {
   } else {
     // We need to use big ints so we implement the bit-mask via the binary string
     // representation rather than using a big integer class
-    std::string mask_bin = Utils::hex2bin(mask); // has 0b prefix while creg_register_ doesn't
-    size_t length = std::min(mask_bin.size() - 2, creg_register_.size()); // -2 to remove 0b
+    std::string mask_bin = Utils::hex2bin(mask, false); // has 0b prefix while creg_register_ doesn't
+    size_t length = std::min(mask_bin.size(), creg_register_.size()); // -2 to remove 0b
```

Comment:
I think the // -2 to remove 0b comment is redundant now and can be removed?

Reply:
Yeah I think so too. I removed it and rebased the commits onto [Qiskit:master](https://github.com/Qiskit/qiskit-aer/commit/33702ebe5c7b3d0cecbac5b7e2ddae7c1dd54dca).